media/inf/wdmaudio.inf

```diff
@@ -164,18 +162,16 @@ StartType      = 3                  ; SERVICE_DEMAND_START
 ErrorControl   = 1                  ; SERVICE_ERROR_NORMAL
 ServiceBinary  = %10%\system32\drivers\drmkaud.sys
 
-
-;; Destination Directory List
 [DestinationDirs]
 WDM.CopyFiles  = 10,system32\drivers ; %SystemRoot%\system32\drivers
 WDM_DRMKAUD.CopyFiles  = 10,system32\drivers
 
-;---------------------------------------------------------------------------------
+;------------------------ STRINGS ---------------------------------------
```

Comment:
Please keep an equal number of dashes before and after the "title" (Here and in the other changes you've made to those sort of comments).

Reply:
ok. I will change that.

Reply:
Done. Words are centered now.